articles/batch-processing-vs-stream-processing/index.md

```diff
@@ -0,0 +1,75 @@
+Organizations are leveraging cloud technologies and big data to promote a data-driven decision making culture, modernize data centers, and improve traditional information technology structure. Cloud computing has transformed the data analytics industry in many aspects.
+
+Particularly, cloud-based technologies ensure proper data management structure. However, this is faced with the challenge of getting large volumes of data processed faster. Batch processing and stream processing are two primary solutions to the faster processing of large data sets.
+
+This article discusses batch processing and stream processing, their benefits, drawbacks, and use cases.
+
+### An overview of batch processing
+
+In batch processing, large data volumes are processed in a batch or a group within a specified time interval. The batch processing system is programmed so that once it is fed with a set of data files as input, it processes the data and then produces a set of data files as output.
+
+You have to wait for a certain amount of raw data to accumulate before running the extract, transform, load (ETL) process in batch processing. [ETL](https://www.webopedia.com/definitions/etl/) represents three database functions (extract, transform, load), which pull data out of a database, transform the data by performing concatenations and calculations, and loads the data into another data warehouse system.
+
+Basically, in batch processing, data is available for analysis after an hour or a few days. Batch ETL is run on a specified schedule, for instance, every 24 hours. You can also set the system to run batch ETL once the data reaches a certain limit.
+
+### An overview of stream processing
+
```

Comment:
How can we solve complex problems using stream processing? Implement machine learning models? Can you elaborate on that?